prow/jobs_config/knative-extensions/kn-plugin-event.yaml

```
@@ -49,10 +49,12 @@ jobs:
       - bash
       - -c
       - |
-        mkdir -p /root/.kube
-        cat /opt/cluster/ci-script > connect.sh
-        chmod +x connect.sh
-        ./connect.sh ${CI_JOB}
+        cat /opt/cluster/vm-script > /tmp/vm.sh
+        cat /opt/cluster/ci-script > /tmp/connect.sh
+        chmod +x /tmp/vm.sh
+        chmod +x /tmp/connect.sh
+        server_vm=$(. /tmp/vm.sh)
+        . /tmp/connect.sh ${CI_JOB} ${server_vm}
```

Those executions could all be simplified like this:

```suggestion
        server_vm="$(sh /opt/cluster/vm-script)"
        source /opt/cluster/ci-script "${CI_JOB}" "${server_vm}"
```

---

Hey @cardil , on ppc64le. We don't have dedicated clusters for Knative upstream , we recreate the clusters for every test we run . I will incorporate these changes u mention